db/migrate/20230912072042_update_mentee_application_state_stages.rb

```diff
@@ -0,0 +1,17 @@
+class UpdateMenteeApplicationStateStages < ActiveRecord::Migration[7.0]
+  def up
+    rejected_applications = MenteeApplicationState.where(status: 6)
```

Comment:
migration to map `:accepted` and `:rejected` to the correct value before casting.

Reply:
oh nice! totally missed this.